ENIGMAsystem/SHELL/Platforms/xlib/XLIBmain.cpp

```diff
@@ -194,14 +198,53 @@ void handleInput() {
 
 }  // namespace enigma
 
+namespace {
+
+int display_get_result = 0;
```

Comment:
You are never assigning this to anything other than zero. I'd replace it with zero.

Reply:
It's actually being assigned within the function. It returns the correct values for each one.